src/pages/_app.page.tsx

```diff
@@ -61,8 +62,14 @@ export default function MyApp({ Component, pageProps }: AppProps) {
   const updateAccountCenter = useAccountCenter()
   const loadWalletFromLocalStorage = useLoadWalletFromLocalStorage()
   const storeWalletsInLocalStorage = useStoreWalletsInLocalStorage()
+  const loadSafeWallet = useLoadSafeWallet()
   const connectedWallets = useWallets()
 
+  // If possible, load Safe wallets
+  useEffect(() => {
+    loadSafeWallet()
+  }, [loadSafeWallet])
+
```

Comment:
Hey @filipvvv , thanks for the PR! Just a question as I haven't had a chance to test this, but won't this prompt all users with the connect modal if not connected when loading a page on jb?

Reply:
@wraeth-eth it won't — [Blocknative's implementation](https://github.com/blocknative/web3-onboard/blob/v2-web3-onboard-develop/packages/gnosis/src/index.ts) checks to see (i) if a user is using a verified Gnosis Safe domain and (ii) if `window.self !== window.top` (verifies that it's embedded).

I originally had these checks but realized they were redundant with `disableModals: true`.